### ISSUE TITLE
Creating worlds for Minecraft folks

### ISSUE THREAD

**Issue by AugustT** [AugustT/CEHCraft#18]
It might be a nice idea to build the worlds of a bunch of famous Minecraft players/commentators. I have had a look at a few forums and here are some potential people.

Joseph Garrett / Stampy Cat - Born in Portsmouth - Uni in Southampton - We could do Southampton
Daniel Middleton / DanTDM - Lives in Wellingborough
Various people / yogscast - The team is based in Bristol

**Comment by GPowney** [comment]
Yes, good idea. Only concern is that given the stalker stories, etc... many Youtubers may prefer we don't map their local area (even though it'll be super broad and people won't be able to pinpoint where they live...). We could always drop their management team an email?!

**Comment by AugustT** [comment]
I got all of these off their Wikipedia pages, so I don't think it is much of a secret!
